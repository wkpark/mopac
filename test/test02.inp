  CHARGE=2 OPEN(4,3) SINGLET ROOT=6 force 
 Methane RHF dication, 6th singlet state (totally symmetric)
  Check that vibrational frequencies are 1342(T) 2488(A) 2554(E) 4619(T) +/-1
  H    
  C    1.2298156  
  H    1.2298156  0    109.471221  
  H    1.2298156  0    109.471221  0   -120.000000  0    2    1    3
  H    1.2298156  0    109.471221  0    120.000000  0    2    1    3

