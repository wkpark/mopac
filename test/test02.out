 *******************************************************************************
 ** FRANK J. SEILER RES. LAB., U.S. AIR FORCE ACADEMY, COLO. SPGS., CO. 80840 **
 *******************************************************************************

                                MNDO CALCULATION RESULTS


 *******************************************************************************
 *          MOPAC:  VERSION  6.00               CALC'D. 31-OCT-90               
 *  SINGLET  - SPIN STATE DEFINED AS A SINGLET
 *
 *
 *
 *                 CHARGE ON SYSTEM =  2
 *
 *
 *
 *  OPEN(N,N)- THERE ARE 4 ELECTRONS IN 3 LEVELS
 *   T=      - A TIME OF  3600.0 SECONDS REQUESTED
 *  DUMP=N   - RESTART FILE WRITTEN EVERY  3600.0 SECONDS
 *  FORCE    - FORCE CALCULATION SPECIFIED
 *  ROOT     - IN A C.I. CALCULATION, ROOT 6 TO BE OPTIMIZED.
 ***********************************************************************040BY040
  CHARGE=2 OPEN(4,3) SINGLET ROOT=6 FORCE
 Methane RHF dication, 6th singlet state (totally symmetric)
  Check that vibrational frequencies are 1342(T) 2488(A) 2554(E) 4619(T) +/-1

    ATOM   CHEMICAL  BOND LENGTH    BOND ANGLE     TWIST ANGLE
   NUMBER  SYMBOL    (ANGSTROMS)     (DEGREES)      (DEGREES)
    (I)                  NA:I          NB:NA:I      NC:NB:NA:I     NA   NB   NC

      1      H       
      2      C         1.22982                                     1
      3      H         1.22982        109.47122                    2    1
      4      H         1.22982        109.47122    -120.00000      2    1    3
      5      H         1.22982        109.47122     120.00000      2    1    3


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         H        0.0000    0.0000    0.0000
     2         C        1.2298    0.0000    0.0000
     3         H        1.6398    1.1595    0.0000
     4         H        1.6398   -0.5797   -1.0041
     5         H        1.6398   -0.5797    1.0041
  H: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       
  C: (MNDO):  M.J.S. DEWAR, W. THIEL, J. AM. CHEM. SOC., 99, 4899, (1977)       
 THERE ARE  1 DOUBLY FILLED LEVELS


          RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =  1

                           NO. OF LEVELS WITH OCCUPANCY 1.333  =  3


            INTERATOMIC DISTANCES
0
                  H  1       C  2       H  3       H  4       H  5
 ------------------------------------------------------------------
     H    1   0.000000
     C    2   1.229816   0.000000
     H    3   2.008280   1.229816   0.000000
     H    4   2.008280   1.229816   2.008280   0.000000
     H    5   2.008280   1.229816   2.008280   2.008280   0.000000


          HEAT OF FORMATION =  832.964491 KCALS/MOLE


          INTERNAL COORDINATE DERIVATIVES

   ATOM  AT. NO.  BOND           ANGLE          DIHEDRAL

     1     H
     2     C    -0.016586
     3     H    -0.015341    -0.002182
     4     H    -0.014415     0.000249    -0.000112
     5     H    -0.014380     0.000126     0.000165


          GRADIENT NORM =   0.03049


          TIME FOR SCF CALCULATION =    0.45


          TIME FOR DERIVATIVES     =    2.19

          MOLECULAR WEIGHT =   16.04



           PRINCIPAL MOMENTS OF INERTIA IN CM(-1)

          A =    4.146970   B =    4.146970   C =    4.146970



           PRINCIPAL MOMENTS OF INERTIA IN UNITS OF 10**(-40)*GRAM-CM**2

          A =    6.750142   B =    6.750142   C =    6.750142


         ORIENTATION OF MOLECULE IN FORCE CALCULATION

    NO.       ATOM         X         Y         Z

     1         1       -1.2298    0.0000    0.0000
     2         6        0.0000    0.0000    0.0000
     3         1        0.4099    1.1595    0.0000
     4         1        0.4099   -0.5797   -1.0041
     5         1        0.4099   -0.5797    1.0041


    FIRST DERIVATIVES WILL BE USED IN THE CALCULATION OF SECOND DERIVATIVES

          ESTIMATED TIME TO COMPLETE CALCULATION =    79.20 SECONDS
 STEP:   1 TIME =     4.83 SECS, INTEGRAL =      4.83 TIME LEFT:   3592.53
 STEP:   2 TIME =     4.72 SECS, INTEGRAL =      9.55 TIME LEFT:   3587.81
 STEP:   3 TIME =     4.59 SECS, INTEGRAL =     14.14 TIME LEFT:   3583.22
 STEP:   4 TIME =     4.82 SECS, INTEGRAL =     18.96 TIME LEFT:   3578.40
 STEP:   5 TIME =     4.87 SECS, INTEGRAL =     23.83 TIME LEFT:   3573.53
 STEP:   6 TIME =     4.73 SECS, INTEGRAL =     28.56 TIME LEFT:   3568.80
 STEP:   7 TIME =     4.94 SECS, INTEGRAL =     33.50 TIME LEFT:   3563.86
 STEP:   8 TIME =     4.76 SECS, INTEGRAL =     38.26 TIME LEFT:   3559.10
 STEP:   9 TIME =     4.43 SECS, INTEGRAL =     42.69 TIME LEFT:   3554.67
 STEP:  10 TIME =     4.52 SECS, INTEGRAL =     47.21 TIME LEFT:   3550.15
 STEP:  11 TIME =     4.46 SECS, INTEGRAL =     51.67 TIME LEFT:   3545.69
 STEP:  12 TIME =     4.45 SECS, INTEGRAL =     56.12 TIME LEFT:   3541.24
 STEP:  13 TIME =     4.87 SECS, INTEGRAL =     60.99 TIME LEFT:   3536.37
 STEP:  14 TIME =     4.43 SECS, INTEGRAL =     65.42 TIME LEFT:   3531.94
 STEP:  15 TIME =     4.53 SECS, INTEGRAL =     69.95 TIME LEFT:   3527.41


           FORCE MATRIX IN MILLIDYNES/ANGSTROM
0
                  H  1       C  2       H  3       H  4       H  5
 ------------------------------------------------------------------
     H    1   3.884040
     C    2   7.610852  27.271856
     H    3   1.519405   7.611042   3.884522
     H    4   1.519409   7.611411   1.519423   3.884489
     H    5   1.519442   7.611465   1.519482   1.519604   3.884544


          HEAT OF FORMATION =  832.964491 KCALS/MOLE


           ZERO POINT ENERGY      36.463 KILOCALORIES PER MOLE


    THE LAST 6 VIBRATIONS ARE THE TRANSLATION AND ROTATION MODES
    THE FIRST THREE OF THESE BEING TRANSLATIONS IN X, Y, AND Z, RESPECTIVELY


           NORMAL COORDINATE ANALYSIS




    ROOT NO.    1           2           3           4           5           6

          1342.86621  1343.10636  1343.38551  2488.47184  2553.94262  2554.11926
  
         1  -0.19449     0.14489     0.29015     0.41962    -0.00013    -0.00018
         2   0.16489    -0.29237     0.25651    -0.00014    -0.07781    -0.41238
         3   0.32259     0.25847     0.08720     0.00012     0.41235    -0.07780
         4  -0.02686     0.02001     0.04007     0.00000     0.00000     0.00000
         5  -0.02038     0.03614    -0.03172     0.00000     0.00000     0.00001
         6  -0.03988    -0.03196    -0.01078     0.00000     0.00000     0.00000
         7   0.07331     0.04633    -0.40862    -0.13973     0.07342     0.38884
         8  -0.24225     0.29652     0.01743    -0.39569    -0.02578    -0.13739
         9   0.32256     0.25849     0.08716    -0.00008    -0.41235     0.07778
        10   0.38701    -0.08156    -0.13452    -0.13984     0.30005    -0.25782
        11  -0.07516    -0.40597    -0.01155     0.19797     0.28988     0.22991
        12  -0.09319     0.06169    -0.37730     0.34253    -0.04497    -0.23815
        13   0.05424    -0.34817    -0.22448    -0.14002    -0.37330    -0.13079
        14   0.39540    -0.02890     0.11560     0.19787    -0.18629     0.31973
        15  -0.07672    -0.19781     0.33139    -0.34253     0.04500     0.23815




    ROOT NO.    7           8           9          10          11          12

          4619.59380  4619.79509  4619.83238    -3.00102    -5.66657    -3.75392
  
         1   0.54776    -0.19517     0.19195     0.26574     0.00000     0.00000
         2   0.07948     0.03195    -0.18974     0.00000     0.26574     0.00000
         3  -0.05089    -0.19765    -0.05299     0.00000     0.00000    -0.26574
         4  -0.10337     0.03683    -0.03622     0.26574     0.00000     0.00000
         5  -0.04371    -0.01757     0.10432     0.00000     0.26574     0.00000
         6   0.02797     0.10866     0.02914     0.00000     0.00000    -0.26574
         7   0.27586    -0.06204    -0.03422     0.26574     0.00000     0.00000
         8   0.32776     0.04601    -0.47283     0.00000     0.26574     0.00000
         9  -0.05086    -0.19768    -0.05304     0.00000     0.00000    -0.26574
        10   0.23061     0.01208     0.16457     0.26574     0.00000     0.00000
        11   0.01931    -0.07987    -0.32930     0.00000     0.26574     0.00000
        12  -0.15514    -0.39138    -0.29465     0.00000     0.00000    -0.26574
        13   0.17763    -0.19380     0.10934     0.26574     0.00000     0.00000
        14   0.09427     0.21124    -0.25126     0.00000     0.26574     0.00000
        15  -0.07643    -0.50820     0.05341     0.00000     0.00000    -0.26574




    ROOT NO.   13          14          15

           -13.92138    30.79518    16.54582
  
         1   0.00000     0.00000     0.00000
         2   0.00780     0.31194    -0.42265
         3  -0.00383     0.42032     0.31371
         4   0.00000     0.00000     0.00000
         5   0.00000     0.00000     0.00000
         6   0.00000     0.00000     0.00000
         7   0.00735     0.29410    -0.39848
         8  -0.00260    -0.10398     0.14088
         9   0.52045    -0.14151    -0.09687
        10  -0.00055    -0.49024    -0.05690
        11   0.44702    -0.10520     0.14755
        12  -0.25831    -0.13940    -0.10842
        13  -0.00680     0.19614     0.45538
        14  -0.45222    -0.10276     0.13422
        15  -0.25831    -0.13940    -0.10842


           MASS-WEIGHTED COORDINATE ANALYSIS




    ROOT NO.    1           2           3           4           5           6

          1342.86621  1343.10636  1343.38551  2488.47184  2553.94262  2554.11926
  
         1  -0.23264     0.17329     0.34702     0.49999    -0.00015    -0.00021
         2   0.19723    -0.34966     0.30678    -0.00017    -0.09272    -0.49137
         3   0.38585     0.30912     0.10429     0.00014     0.49133    -0.09270
         4  -0.11090     0.08263     0.16542    -0.00001    -0.00002    -0.00002
         5  -0.08415     0.14922    -0.13096     0.00000     0.00000     0.00004
         6  -0.16467    -0.13194    -0.04450    -0.00001    -0.00001     0.00000
         7   0.08768     0.05540    -0.48870    -0.16649     0.08748     0.46332
         8  -0.28976     0.35463     0.02085    -0.47149    -0.03072    -0.16371
         9   0.38582     0.30915     0.10424    -0.00010    -0.49134     0.09268
        10   0.46291    -0.09755    -0.16089    -0.16662     0.35752    -0.30721
        11  -0.08990    -0.48553    -0.01381     0.23589     0.34540     0.27395
        12  -0.11146     0.07378    -0.45124     0.40814    -0.05358    -0.28377
        13   0.06488    -0.41640    -0.26847    -0.16684    -0.44480    -0.15584
        14   0.47294    -0.03456     0.13825     0.23578    -0.22197     0.38097
        15  -0.09177    -0.23658     0.39633    -0.40814     0.05362     0.28377




    ROOT NO.    7           8           9          10          11          12

          4619.59380  4619.79509  4619.83238    -3.00102    -5.66657    -3.75392
  
         1   0.62086    -0.22044     0.22034     0.25066     0.00000     0.00000
         2   0.09008     0.03608    -0.21781     0.00000     0.25065     0.00000
         3  -0.05768    -0.22323    -0.06083     0.00000     0.00000    -0.25065
         4  -0.40447     0.14361    -0.14353     0.86527     0.00000     0.00000
         5  -0.17101    -0.06849     0.41337     0.00000     0.86527     0.00000
         6   0.10945     0.42367     0.11547     0.00000     0.00000    -0.86527
         7   0.31267    -0.07007    -0.03928     0.25065     0.00000     0.00000
         8   0.37150     0.05196    -0.54276     0.00000     0.25066     0.00000
         9  -0.05765    -0.22327    -0.06088     0.00000     0.00000    -0.25065
        10   0.26138     0.01364     0.18891     0.25065     0.00000     0.00000
        11   0.02188    -0.09021    -0.37800     0.00000     0.25065     0.00000
        12  -0.17585    -0.44204    -0.33822     0.00000     0.00000    -0.25065
        13   0.20133    -0.21889     0.12551     0.25065     0.00000     0.00000
        14   0.10685     0.23858    -0.28842     0.00000     0.25065     0.00000
        15  -0.08663    -0.57398     0.06131     0.00000     0.00000    -0.25065




    ROOT NO.   13          14          15

           -13.92138    30.79517    16.54583
  
         1   0.00000     0.00000     0.00000
         2   0.00867     0.36495    -0.49167
         3  -0.00426     0.49174     0.36493
         4   0.00000     0.00000     0.00000
         5   0.00000     0.00000     0.00000
         6   0.00000     0.00000     0.00000
         7   0.00817     0.34408    -0.46355
         8  -0.00289    -0.12165     0.16389
         9   0.57870    -0.16556    -0.11269
        10  -0.00061    -0.57354    -0.06619
        11   0.49705    -0.12307     0.17165
        12  -0.28722    -0.16309    -0.12612
        13  -0.00757     0.22947     0.52974
        14  -0.50283    -0.12023     0.15613
        15  -0.28722    -0.16309    -0.12612
1

          DESCRIPTION OF VIBRATIONS


 VIBRATION   1            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     1342.87       H 1 --  C 2           25.2% ( 92.8%)               3.7%
 T-DIPOLE   1.4020       C 2 --  H 3           25.1%                        4.2%
 TRAVEL     0.1611       C 2 --  H 5           24.9%                        6.0%
 RED. MASS  0.9667       C 2 --  H 4           24.8%                        6.8%

 VIBRATION   2            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     1343.11       C 2 --  H 4           25.6% ( 93.6%)               1.1%
 T-DIPOLE   1.4024       H 1 --  C 2           25.5%                        1.8%
 TRAVEL     0.1611       C 2 --  H 5           24.6%                        9.3%
 RED. MASS  0.9666       C 2 --  H 3           24.3%                       14.3%

 VIBRATION   3            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     1343.39       C 2 --  H 3           25.6% ( 93.5%)               1.2%
 T-DIPOLE   1.4020       C 2 --  H 5           25.5%                        1.7%
 TRAVEL     0.1612       C 2 --  H 4           24.5%                        9.9%
 RED. MASS  0.9662       H 1 --  C 2           24.3%                       13.4%

 VIBRATION   4            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     2488.47       C 2 --  H 3           25.0% ( 70.2%)             100.0%
 T-DIPOLE   0.0009       C 2 --  H 5           25.0%                      100.0%
 TRAVEL     0.1640       C 2 --  H 4           25.0%                      100.0%
 RED. MASS  0.5039       H 1 --  C 2           25.0%                      100.0%

 VIBRATION   5            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     2553.94       C 2 --  H 4           25.0% ( 35.6%)               0.0%
 T-DIPOLE   0.0005       C 2 --  H 5           25.0%                        0.0%
 TRAVEL     0.1618       C 2 --  H 3           25.0%                        0.0%
 RED. MASS  0.5039       H 1 --  C 2           25.0%                        0.0%

 VIBRATION   6            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     2554.12       C 2 --  H 3           25.0% ( 35.6%)               0.0%
 T-DIPOLE   0.0006       H 1 --  C 2           25.0%                        0.0%
 TRAVEL     0.1618       C 2 --  H 4           25.0%                        0.0%
 RED. MASS  0.5039       C 2 --  H 5           25.0%                        0.0%

 VIBRATION   7            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     4619.59       H 1 --  C 2           47.8% ( 65.1%)              91.6%
 T-DIPOLE  12.6570       C 2 --  H 3           29.6%                       67.3%
 TRAVEL     0.0800       C 2 --  H 4           13.6%                       23.6%
 RED. MASS  1.1393       C 2 --  H 5            9.0%                        1.6%

 VIBRATION   8            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     4619.80       C 2 --  H 5           52.1% ( 68.0%)              95.6%
 T-DIPOLE  12.6580       C 2 --  H 4           25.6%                       59.2%
 TRAVEL     0.0800       H 1 --  C 2           13.6%                       23.6%
 RED. MASS  1.1393       C 2 --  H 3            8.7%                        0.4%

 VIBRATION   9            ATOM PAIR     ENERGY CONTRIBUTION              RADIAL
 FREQ.     4619.83       C 2 --  H 3           36.6% ( 57.0%)              78.6%
 T-DIPOLE  12.6582       C 2 --  H 4           35.8%                       77.4%
 TRAVEL     0.0800       C 2 --  H 5           13.9%                       24.9%
 RED. MASS  1.1393       H 1 --  C 2           13.6%                       23.6%
1



 TOTAL CPU TIME:            74.03 SECONDS

 == MOPAC DONE ==
